### PR TITLE
Usage for the consistency while referring to API

### LINKED ISSUE #1109: Consistency in using 'the' when referring to APIs
## Description

Need to check docs for consistency in use of 'the' when referring to APIs. 

For example, this page:

https://developer.nexmo.com/tutorials/sending-whatsapp-messages-with-messages-api

Does not use 'the' in the title.

### PR DESCRIPTION
**Pull request by arshadkazmi42**
Fixes #1109 
----------

## Description

Consistency in using `the` when referring to API

## Deploy Notes

N/A

### CODE REVIEW

File: _open_api/definitions/external-accounts.yml [hunk 1]
```diff
@@ -769,7 +769,7 @@ x-groups:
   account:
     name: "Account"
     order: 2
-    description: An external-account used as the `from` field in Messages API and Dispatch API
+    description: An external-account used as the `from` field in  Messages API and Dispatch API
```
Comment (mheap) [inline _open_api/definitions/external-accounts.yml]: Should `the` be here?

Comment (arshadkazmi42) [reply _open_api/definitions/external-accounts.yml]: I added it before, then removed, as after adding `the` it becomes `the Messages API and the Dispatch API`

Comment (arshadkazmi42) [reply _open_api/definitions/external-accounts.yml]: Should I add it back?

Comment (mheap) [reply _open_api/definitions/external-accounts.yml]: How about `in the Messages API and Dispatch API`?

Comment (arshadkazmi42) [reply _open_api/definitions/external-accounts.yml]: This sounds better. Let me make that change right away

**Pull request merged**